### PR TITLE
Support Null safety

### PR DESCRIPTION
**Pull request by leynier**

### CODE REVIEW

**Comment by leynier** [comment]
Fix #11

**Comment by hmawla** [comment]
This looks good

File: lib/src/method_channel.dart [hunk 1]
```diff
@@ -8,7 +8,8 @@ class FlutterDeviceLocaleMethodChannelPlugin
     extends FlutterDeviceLocalePlatform {
   @override
   Future<List<String>> deviceLocales() async {
-    final List<dynamic> result = await _channel.invokeMethod('deviceLocales');
+    final result =
+        await (_channel.invokeMethod('deviceLocales') as Future<List<dynamic>>);
```
Comment (nohli) [inline lib/src/method_channel.dart]: This should be

`final result = (await _channel.invokeMethod('deviceLocales') as List<dynamic>);`

Fixes https://github.com/bratan/flutter_translate/pull/63#issuecomment-815762314

Comment (leynier) [reply lib/src/method_channel.dart]: Thanks very much

**Pull request closed without merging**